wg-ecosystem/incidents/2020-05-01.md

```diff
@@ -0,0 +1,30 @@
+# Post-incident report for 1 May 2020
+
+## What's happened
+
+23 April after the default module updating, the website is unable to stable deploy. The fast reaction team disables the automatic deploys and reverts the website to the last good commit.
+
+## First investigation (around 0:00 UTC May 1)
+
+1 May we started the investigation of that errors. We created 2 deploys, the first is `master` branch build with the latest changes, the second deploy is with a potential fix but that also failed. This investigation doesn't give more information since we are not able to find the root cause of errors. The big problem is that the crash only reproducible on `master` deploy, what's creates big problems with debugging.
+
+## Further Investigation (Morning of May 1 for Europe)
+
+The future investigation continued by starting the isolated Heroku instance of a website. It's been created to read build logs, and easy debugging of error. The build log gives more information about what's Node version and dependencies we take under the building.
+
+The build logs give info that we use Node 14 when deploying from `master`, what we not test, and not plan to use for production deploys currently. The problem here it's that node version range says that we can use Node >= 12.
+
+We don't took that version in the first since thinking the Heroku picks the Node 12 as they LTS but that not true. The errors that are we have seen 23 April it's that the deploy Heroku takes the Node 14 when it's not ready and tested on our CI processes.
+
+At around 19:45 UTC, we merge the hotfix patch what tested in the isolated instance and don't give any errors or crashes. This commit is successfully deployed, and we don't see any degradation after them.
```

Comment:
```suggestion
At around 19:45 UTC, we merged the hotfix patch that was tested in the isolated instance. This commit was successfully deployed, and there has not been any further downtime related to this problem since.
```